### PR TITLE
Feature/tr 920 audio interaction pci

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/TR-920

- composer file: https://github.com/oat-sa/kitchen-playground/blob/test/jus1/files/delivery/composer.json
- if you using existing instance, you will need to run migration `sudo -u www-data php index.php '\oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\qtiItemPci\migrations\Version202111091818551465_qtiItemPci' -e qtiItemPci`
- go to authoring and create an item with audio recording PCI, change the language to Arabic (RTL)
- observe position aligned to the righ
Screenshot 2021-11-09 at 19 32 12
t:

### CODE REVIEW

File: views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss [hunk 1]
```diff
@@ -158,6 +200,14 @@ $playback: whiten($info, .3);
       height: 10px;
       border-radius: 5px;
```
Comment (marpesia) [inline views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss:202]: Would you please check this?
I think it is working fine without this code. The `dir` param is already changing the progressBar direction.
With CSS code
image
Without it
image

File: views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss [hunk 2]
```diff
@@ -158,6 +200,14 @@ $playback: whiten($info, .3);
...
 
+      @include ltr {
+        transform: scaleX(1);
+      }
+
+      @include rtl {
+        transform: scaleX(-1);
+      }
+
```
Comment (marpesia) [inline views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss:210]: Would you please check this?
I think it is working fine without this code. The `dir` param is already changing the progressBar direction.
With CSS code
image
Without it
image

**Review by marpesia — CHANGES_REQUESTED** [review]

File: views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss (hunk 2 above)
Comment (meefox) [reply views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss:210]: a bar direction should stay the same as for RTL and LTR user, only aligment is diffiernt

**Review by marpesia — APPROVED** [review]
LGTM

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ]  Commits messages are meaningful
- [ ]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by krampstudio — APPROVED** [review]
Tested in FF and Chome, successfully.
 Tested the migration script successfully too. 

Good job :+1: 

- [ ] New code is covered by tests (CSS only)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] <s>New code is not subject to concurrency issues (if applicable)</s>
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**